src/customizations/volto/components/manage/Blocks/Image/Edit.jsx

```diff
@@ -117,7 +117,12 @@ class ImageEdit extends Component {
           >
             <Image
               loading="lazy"
-              src={data}
+              // TODO: Improve this, please
+              src={
+                data.image_scales
+                  ? data
+                  : `${flattenToAppURL(data.url)}/@@images/image`
+              }
```

Comment:
@sneridagh I am not following why this is needed. We should be getting the image_scales from the catalog in edit mode also. Steps to reproduce?

Reply:
Oh, when you upload, rather than selecting an existing image. Hmm, let me check if we had a solution for this in our client project.